spec/sidekiq_flow/adapters/set_storage_spec.rb

```diff
@@ -0,0 +1,480 @@
+require 'spec_helper'
+
+RSpec.shared_examples '.start_workflow common' do
+  it 'should store the workflow in redis' do
+    expect {
+      subject
+    }.to change { $redis.keys.count }.from(0).to(2)
```

Comment:
there is a `redis`  variable that is a reference of `$redis`, don't use the global one.

see: `let(:redis) { $redis }`

Reply:
It's in the `RSpec.shared_examples` block, so no `redis` reference yet. Do I create one ?

Reply:
shared examples inherit variables from the context that included them.